sdcm/sct_config.py

```diff
@@ -386,6 +386,9 @@ class SCTConfiguration(dict):
         dict(name="use_mgmt", env="SCT_USE_MGMT", type=boolean,
              help="When define true, will install scylla management"),
 
+        dict(name="parallel_node_operations", env="SCT_PARALLEL_NODE_OPERATIONS", type=boolean,
+             help="When defined true, will run node operations in parallel. Supported operations: startup"),
```

Comment:
You already have the code for add node and decommission, aren't you?
I think you can add it to this PR.

Reply:
decommission / grow is more complicated and requires some work (e.g. in my code it doesn't work yet with multi-az).
I wanted to make this one in and later add more complicated ones (and easier to revert in case of issues).